__tests__/components/forecast-summary.test.jsx

```diff
@@ -1,52 +1,27 @@
 import React from 'react';
 import Enzyme from 'enzyme';
```

Comment:
Try using a named import. `import { shallow } form 'enzyme'` would let you just use `shallow()` everywhere else in your code.